vaadin-text-field-flow-demo/src/main/java/com/vaadin/flow/component/textfield/demo/TextFieldView.java

```diff
@@ -356,6 +421,52 @@ private void textAreaMinHeight() {
         addCard("Text Area", "Minimum height", textArea);
     }
 
+    private void textAreaHelperText() {
+        Div div = new Div();
+        // begin-source-example
+        // source-example-heading: Text area Helper Text and Component
+        TextArea textAreaHelperText = new TextArea("Overview");
+        textAreaHelperText.setPlaceholder("Write here ...");
+        textAreaHelperText
+              .setHelperText("Short description of your current position");
+
+        TextArea textAreaHelperComponent = new TextArea("Feedback");
+        textAreaHelperComponent.setPlaceholder("Write here ...");
+        textAreaHelperComponent.setHelperComponent(new Span(
+              "Here you can share what you've liked and what can be improved in the next lesson"));
+
+        add(textAreaHelperText, textAreaHelperComponent);
+        // end-source-example
+
+        textAreaHelperText.setId("text-area-with-helper-text");
+        textAreaHelperText.getStyle().set("margin-right","20px");
+        div.add(textAreaHelperText, textAreaHelperComponent);
+        addCard("Text Area", "Text area Helper Text and Component", div);
+    }
+
+    private void textAreaHelperTextVariant() {
+        Div div = new Div();
+        // begin-source-example
+        // source-example-heading: TextArea Helper Variant
+        TextArea textAreaBelow = new TextArea("Helper Text below the field");
+        textAreaBelow.setPlaceholder("Write here ...");
+        textAreaBelow.setHelperText("Helper Text");
+
+        TextArea textAreaAbove = new TextArea("Helper Text above the field");
+        textAreaAbove.setPlaceholder("Write here ...");
+        textAreaAbove.setHelperText("Helper Text");
+        textAreaAbove.addThemeVariants(TextAreaVariant.LUMO_HELPER_ABOVE_FIELD);
```

Comment:
Maybe just the helper text would work better here. I feel that the label and the helper text having both "Helper text" on it, might be confusing.

Reply:
Done :)